stores/bookstore.ts

```diff
@@ -1,3 +1,5 @@
+import { getTimestampRoundedToMinute } from '../utils'
```

Comment:
not auto import?

Reply:
`getTimestampRoundedToMinute` is not found error randomly shows up, not sure if related to dev vs prod or ssr vs client